packages/components/src/components/audio/audio.ts

```diff
@@ -251,9 +251,14 @@ export default class SdAudio extends SolidElement {
     }
   }
 
+  private handleTranscriptDrawer() {
```

Comment:
Should we give it a clearer name?

```suggestion
  private handleTranscriptDrawerToggle() {
```